etc/inc/allow-pandoc.inc

```diff
@@ -0,0 +1,9 @@
+# This file is overwritten during software install.
+# Persistent customizations should go in a .local file.
+include allow-pandoc.local
+
+noblacklist ${PATH}/pandoc
+noblacklist /usr/bin/pandoc
+noblacklist /usr/lib/ghc-*
+noblacklist /usr/lib/*pandoc*
```

Comment:
Never `blacklist`ed. None of them.

Reply:
Very true. The whitelist directive is enough.

I use pandoc to display html email in neomutt, I thought I'd share it. Might share the neomutt.profile as well.

If one does not whitelist `/usr/share/pandoc` the following happens:
```
$ neomutt
Could not find data file /usr/share/pandoc/data/abbreviations
Press any key to continue...
```